src/test/testFixtures/diagnostics/yaml/1.yml

```diff
@@ -0,0 +1,6 @@
+---
+- name:
+  hosts: localhost
+  tasks:
+    - debug: msg="foo: bar"
```

Comment:
@priyamsahoo based on what I read from linter, I suspect you forgot to add some quotes on that string?

Reply:
Actually, I want to have this like this so that I validate the diagnostic messages that I get from the extension. That's the purpose of this particular test.

But the problem is, even if I disable yamllint for this file, the I do not get valid message from `ansible-playbook --syntax-check`